shared/store/configure-store.js

```diff
@@ -81,12 +81,23 @@ const loggerMiddleware: any = enableStoreLogging ? createLogger({
   logger,
 }) : null
 
+let lastError = new Error('')
+
 const errorCatching = store => next => action => {
   try {
     return next(action)
   } catch (error) {
+    // Don't let the same error keep getting caught
+    if (lastError.message === error.message) {
```

Comment:
this is to stop the error handler throwing errors. I was getting stuck in a loop due to HMR and theoretically this could happen in prod